src/appShell/App/PortalHeader.tsx

```diff
@@ -139,7 +146,7 @@ export default class PortalHeader extends React.Component<
                                 <div className="identity">
                                     <Dropdown id="dat-dropdown">
                                         <Dropdown.Toggle className="btn-sm">
-                                            Logged in as{' '}
+                                            Logged in as +
```

Comment:
@pvannierop did you mean to put that plus sign there (just before the username) 
result:
![image](https://github.com/cBioPortal/cbioportal-frontend/assets/186521/245537dd-46b7-49cd-bf20-4f08408c1ce0)

Reply:
Eh no, most definitely not. I will revert. Must have been a accidental keyboard press?